arelle/plugin/validate/ROS/PluginValidationDataExtension.py

```diff
@@ -0,0 +1,87 @@
+"""
+See COPYRIGHT.md for copyright information.
+"""
+
+from __future__ import annotations
+import regex as re
+from arelle import ModelDocument
+from dataclasses import dataclass
+from arelle.ModelXbrl import ModelXbrl
+
+from arelle.utils.PluginData import PluginData
+
+taxonomyReferences = [
```

Comment:
Pre-existing, but we should eventually conform code style to the conventions we use elsewhere, such as upper snake case for constant values, e.g. `TAXONOMY_REFERENCES`.